### DISCUSSION TITLE
Programmable Search Engine / Google Adsense Search

### DISCUSSION THREAD

**Discussion by beauchette** [AmauriC/tarteaucitron.js#1218] [Q&A]
Bonjour,

Il est possible de configurer un programmable search Engine avec tarteaucitron mais seulement en utilisant le service Google Adsense Search. Ca me pose probleme, parce qu'en temps que "non-profit", j'utilise Programmable search engin sans publicité. Et si, effectivement, google utilise des cookies pour la recherche, il n'est pas question de suivi pour de la pub.

Or, quand un utilisateur arrive sur le site, et qu'il regarde la liste des cookies, ca lui propose de supprimer Adsense, et j'avoue que moi meme en tant qu'utilisateur, j'aurais envie de dire non, me privant alors de la recherche.

Ne serait-il pas judicieux de rajouter un service pour ce cas ? Je peux d'ailleurs le faire si ca semble logique

**Comment by AmauriC** [comment]
Bonjour,

Sur quel site je peux voir l'intégration du formulaire de recherche?
S'il est possible de le différencier d'un "adsense search" je pourrai créer un deuxième service avec un nom sans adsense ;)

**Reply by beauchette** [reply]
Je me permets de vous envoyer un lien par email (celui que vous indiquez sur GitHub)

**Reply by AmauriC** [reply]
Merci. J'ai regardé et le code est le même donc impossible côté tarteaucitron de différencier un formulaire avec ou sans adsense.
Mais si vous êtes sûr qu'il n'y a pas de tracking, vous pouvez ne pas l'inclure sur tarteaucitron et le charger directement.

**Reply by beauchette** [reply]
Ah, très bien. Je vais quand même vérifier ce que ça met comme cookies et me faire un service juste pour notre site de manière à préciser que c'est pour la recherche et en spécifiant bien l'url de google concernant d'éventuels cookies ou données, j'ai des doutes sur le fait que google ne récolte rien sur ces recherches.
